README.md

```diff
@@ -1,2 +1,37 @@
 # js-assertion-cheat-sheet
+
 This repository contains examples of various, often used, and tricky assertions in the common JS assertion tools.
+
+## CI/CD
+
+### Feature Branch deployment
+
+> [!IMPORTANT]  
+> Before deploying any feature branches, make sure that the AWS CDK Stack is already deployed.
```

Comment:
How can we check it?

Reply:
I have added a sentence about how we can do it currently. An improvement of this current flow would be to implement status badges that I mentioned in this comment: https://github.com/Idea-Pool/js-assertion-cheat-sheet/pull/12#issuecomment-2081527017